CMakeLists.txt

```diff
@@ -285,7 +291,8 @@ if(BUILD_JAVA)
         #Must be set here - see https://cmake.org/cmake/help/v3.3/command/set_source_files_properties.html
         set_source_files_properties(${PROTOBUF_GENERATED_JAVA_SRCS} PROPERTIES GENERATED True)
         #Protobuf files are generated by Maven
-        set(MAVEN_PROTOBUF_REGENERATE_ARGS "-Dprotoc.filepath=${PROTOBUF_PROTOC_EXECUTABLE}" "-Dprotobuf.regenerate=true")
+        set(MAVEN_PROTOBUF_REGENERATE_ARGS "-Dprotoc.filepath=${PROTOBUF_PROTOC_EXECUTABLE}" "-Dprotobuf.regenerate=true"
```

Comment:
-Dprotobuf.regenerate=true is no needed anymore since the default process regenerate them everytime that it's executed.

Reply:
Done